source/fundamentals/write-data.txt

```diff
@@ -0,0 +1,146 @@
+.. _entity-framework-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, code example, write
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+{+framework-core+} allows you to work with data in your application without
+explicitly running database commands. You can perform write operations to
```

Comment:
I think the "Perform write operations" sentence makes things less clear.

Reply:
Good call, I think this section got a little jumbled through the edits. I simplified that sentence